_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -56,6 +60,8 @@ Implementation of [Presentational Roles Conflict Resolution][] varies from one b
 
 ## Background
 
+The presence of prohibited ARIA attributes is often the result of a developer using an incorrect role, or a misunderstanding of the attribute. These attributes are ignored by browsers and other assistive technologies. This often means that a state or property which should exist is missing. This can cause issues under [success criterion 1.3.1 Info and Relationships][sc131] or [4.1.2 Name, Rule Value][sc412].
```

Comment:
You have the reasoning for the secondary requirements like this in most of the rules. For now its fine, but I would prefer to make it more explicit in the future. One option is another sub-heading for the background section.